src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -59,6 +61,33 @@ def dcos_browse(name, resource_group_name):
 
     return
 
+def dcos_install():
```

Comment:
You can the acs/_params.py file to define what install_location should be.
This means that when this method is called, the right install_location will be passed in.

So for example, you can remove the install locations that have been defined here and add the following lines to the `acs/_params.py` file:
```
import platform

def _get_default_install_location():
    system = platform.system()
    if system == 'Windows':
        return 'C:\Program Files\dcos.exe'
    if system == 'Linux':
        return '/usr/local/bin/dcos'
    if system == 'Darwin':
        return '/usr/local/bin/dcos'
    else:
        return None

register_cli_argument('acs dcos install', 'install_location', options_list=('--install-location',),
                      default=_get_default_install_location())
```

This means when the user runs `az acs dcos install -h`, they'll see:
```
Arguments
    --install-location                                        : Default: /usr/local/bin/dcos.
```
with the default being computed automatically.